pkg/asset/quota/quota.go

```diff
@@ -126,6 +126,10 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to get cloud info")
```

Comment:
What is an example of an error that we do not want the installer to tolerate?

Reply:
https://prow.ci.openshift.org/view/gcs/origin-ci-test/logs/release-openshift-ocp-installer-e2e-openstack-4.8/1369468189917843456

 ```level=fatal msg=failed to fetch Cluster: failed to fetch dependency of "Cluster": failed to generate asset "Platform Quota Check": failed to get cloud info: failed to generate OpenStack cloud info: failed to load Quota: failed to load quota limits: failed to get network quota records: failed to get QuotaSets from OpenStack Network API: json: cannot unmarshal string into Go struct field QuotaDetail.quota.router.reserved of type int ```

Reply:
Isn't that an example of an error that we **do** want the installer to tolerate? I am looking for whether there are indeed errors that can come from `GetCloudInfo` that we want to cause the installation to fail. For example, if there is an error creating one of the service clients, do we want to fail installation then?

Reply:
Right, we may also want to tolerate errors creating clients in which case `GetCloudInfo()` won't error at all, however I'm not sure it's going to be that useful: if we fail to create a client in `GetCloudInfo()` we're likely get the same error later on during installation as it would indicate an issue with the underlying cloud. This is probably something we should error on the earliest.